src/passes/analysis/vmodule.cpp

```diff
@@ -10,7 +10,7 @@ namespace VerilogNamespace {
 CoreIRVModule::CoreIRVModule(VModules* vmods, Module* m) : VModule(vmods) {
   Type2Ports(m->getType(),this->ports);
   assert(m->hasDef());
-  this->modname = m->getNamespace()->getName() + "_" + m->getLongName();
+  this->modname = m->getLongName();
```

Comment:
Does this mean we're not prepending the namespace names to the module names anymore?  Should this be checking that `getNamespace()` is not global and only prepending the namespace if it isn't? Or is that logic already being handled elsewhere?

Reply:
The logic (should be) if you specifiy Metadata.verilog.prefix, it will prepend the specified prefix. If you do not specify, it will just use the module name.

Reply:
Okay, that logic is broken for modules which have a CoreIR definition (but not for modules which have a verilog definition). I do not think that this case is being hit in any of your tests, so this PR will fix your issues.

Reply:
Got it